usaspending_api/common/helpers/s3_helpers.py

```diff
@@ -0,0 +1,52 @@
+import boto3
+import io
+import logging
+import re
+
+from typing import Optional, List
+
+from usaspending_api import settings
+
+logger = logging.getLogger("script")
+
+
+def access_s3_object_list(
+    bucket_name: str, regex_pattern: Optional[str] = None
+) -> List["boto3.resources.factory.s3.ObjectSummary"]:
+    """Find all S3 objects in provided bucket.
+
+    If regex is passed, only keys which match the regex are returned
+    """
+
+    logger.info("Gathering all deleted transactions from S3")
+    try:
+        bucket = get_s3_bucket(bucket_name=bucket_name)
+        bucket_objects = list(bucket.objects.all())
+    except Exception:
+        msg = (
+            "Verify settings.USASPENDING_AWS_REGION and settings.DELETED_TRANSACTION_JOURNAL_FILES are correct "
+            "or env variables: USASPENDING_AWS_REGION and DELETED_TRANSACTION_JOURNAL_FILES are set"
```

Comment:
This message isn't very reusable. Would it be possible to use the bucket name to construct this string?

Reply:
good catch